From 5a068df1b69ffd81c8858dfeeade6ffc016000ba Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Mon, 26 Oct 2026 09:38:30 -0300
Subject: [PATCH] configure: Fix have-no-error-execstack substitution

Commit a8455a0b7f9 renamed the configure variable to
libc_cv_test_no_error_execstack, but config.make.in still substitutes
libc_cv_no_error_execstack.

Checked on x86_64-linux-gnu.

(cherry picked from commit ac51044ef6ff50eb697bfc20b180209aaafae389)
---
 config.make.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.make.in b/config.make.in
index 0ddc24a..8b467a2 100644
--- a/config.make.in
+++ b/config.make.in
@@ -53,7 +53,7 @@ c++-bits-std_abs-h = @CXX_BITS_STD_ABS_H@
 enable-werror = @enable_werror@
 
 have-z-execstack = @libc_cv_z_execstack@
-have-no-error-execstack = @libc_cv_no_error_execstack@
+have-no-error-execstack = @libc_cv_test_no_error_execstack@
 have-protected-data = @libc_cv_protected_data@
 have-insert = @libc_cv_insert@
 have-glob-dat-reloc = @libc_cv_has_glob_dat@
